frappe/desk/form/save.py

```diff
@@ -22,6 +22,7 @@ def savedocs(doc, action):
 	# action
 	doc.docstatus = {
 		"Save": DocStatus.draft(),
+		"Save as draft": DocStatus.draft(),
```

Comment:
Why is this required? What you're trying to achieve is just a cosmetic change in UI. IMO nothing in server-side code needs to be touched.

Reply:
You’re right and actually I’m really happy you’re being so strict in frappe/frappe, so I’ll try harder avoiding it! 👍

Reply:
You were so right: it‘s been totally unnecessary.
Thank you for pointing it out to me!